### DISCUSSION TITLE
deep clone

### DISCUSSION THREAD

**Discussion by YueLiXing** [rayepps/radash#396] [Ideas]
we need deep clone method

**Comment by f754699** [comment]
附议

**Comment by zrekekeq** [comment]
附议

**Comment by NickBoomBoom** [comment]
need deep clone
<img width="770" alt="image" src="https://github.com/sodiray/radash/assets/29225479/972db50a-fbaf-43f9-a772-bdca509163dc">

**Reply by xmsz-stu** [reply]
它现在clone后目前是可读不可写
